2-mandatory/5-writing-tests.js

```diff
@@ -34,42 +34,59 @@ function convertScoreToGrade(score) {
   The first test has been written for you. You need to fix the test so that it
   passes.
 */
+
 test("a score of 83 is grade A", () => {
-  expect(convertScoreToGrade(83), "Z");
+  expect(convertScoreToGrade(83)).toEqual ("A");
 });
 
 /*
   The rest of the tests have comments describing what to test and you need to
   write a matching test
 */
 
-test.skip("a score of 71 is grade B", () => {
-  /* Remove the .skip above, then write the test body. */
+
+
+  // * Remove the .skip above, then write the test body. *
+  test("a score of 71 is grade B", () => {
+    expect(convertScoreToGrade(71)).toEqual ("B");
+  
 });
-/*
-  Write a test that checks a score of 68 is grade C
-*/
 
+
+//  Write a test that checks a score of 68 is grade C
+  test("a score of 68 is grade C", () => {
+   expect(convertScoreToGrade(68)).toEqual ("C");
+
+});
 /*
   Write a test that checks a score of 55 is grade D
-*/
-
+*/test("a score of 68 is grade D", () => {
+  expect(convertScoreToGrade(55)).toEqual ("D");
+});
 /*
   Write a test that checks a score of 68 is grade C
-*/
+*/test("a score of 68 is grade C", () => {
+  expect(convertScoreToGrade(68)).toEqual ("C");
+})
 
 /*
   Write a test that checks a score of 55 is grade D
-*/
-
```

Comment:
The tests above look good.
But below - it looks like you might be missing `toEqual`?